templates/pioneer-citp/news-disinfo-study/news-disinfo-study.1.schema.json

```diff
@@ -0,0 +1,86 @@
+{
+    "mozPipelineMetadata": {
+        "bq_metadata_format": "pioneer"
+    },
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "type": "object", "properties": {
+        "WebScience.Measurements.PageNavigation": { "type": "object", "properties": {
```

Comment:
Just a note that the schema fields are normalized for use in BigQuery, so this will end up becoming a column called `web_science_measurements_page_navigation`.

Reply:
Should be fine, thanks for the heads up